common/build-style/go.sh

```diff
@@ -36,6 +36,7 @@ do_build() {
 		go install -p "$XBPS_MAKEJOBS" -mod="${go_mod_mode}" -x -tags "${go_build_tags}" -ldflags "${go_ldflags}" ${go_package}
 	else
 		# Otherwise, build using GOPATH
+		export GO111MODULE=off
```

Comment:
I had thought of using `GO111MODULE=auto` in the entire environment, but this might work just as well, I guess.